Documentation/Configuration/BackendEntryPoint/Index.rst

```diff
@@ -0,0 +1,188 @@
+
+Now point your browser to :samp:`https://example.org/my-specific-path` to
+log into the TYPO3 backend.
+
+
+..  _backend-entry-point-specific-subdomain:
+
+Use a distinct subdomain
+------------------------
+
+..  code-block:: php
+    :caption: config/system/additional.php
+
+    $GLOBALS['TYPO3_CONF_VARS']['BE']['entryPoint'] = 'https://my-backend-subdomain.example.org';
+    $GLOBALS['TYPO3_CONF_VARS']['SYS']['cookieDomain'] = '.example.org';
+
+Now point your browser to `https://my-backend-subdomain.example.org/` to log
+into the TYPO3 backend.
+
+Note that the :php:`$GLOBALS['TYPO3_CONF_VARS']['SYS']['cookieDomain']` is
+necessary, so that backend users can preview website pages or use the admin
+panel.
+
+..  _backend-entry-point-legacy-free:
+
+Legacy-free installation
+------------------------
+
+The legacy entry point `/typo3/index.php` is no longer needed and deprecated in
+favor of handling all backend and frontend requests with :samp:`/index.php`. The
+entry point is still in place in case webserver configuration has not been
+adapted yet and the maintenance and emergency tool is still available via
+:samp:`/typo3/install.php` in order to work in edge cases like broken web server
+routing.
+
+In Composer mode is an additional configuration option for the deactivation of
+the legacy entry point which can be defined in the project's
+:file:`composer.json`.
+
+..  code-block:: json
+    :caption: /path/to/project/composer.json (Excerpt)
+
+    {
+        "extra": {
+            "typo3/cms": {
+                "install-deprecated-typo3-index-php": false
+            }
+        }
+    }
+
+..  _backend-entry-point-migration:
+
+Migration
+=========
+
+A silent update is in place which automatically updates the webserver
+configuration file when accessing the install tool, at least for Apache and
+Microsoft IIS webservers.
+
+..  attention::
+    The silent update does not work, if you are not using the default
+    configuration, which is shipped with Core and automatically applied during
+    the TYPO3 installation process, as basis.
+
+If you use a custom web server configuration you may adapt as follows.
+
+Apache configuration
+--------------------
+
+It is most important to rewrite all `typo3/*` requests to `/index.php`, but also
+`RewriteCond %{REQUEST_FILENAME} !-d` should be removed in order for a request
+to `/typo3/` to be directly served via `/index.php` instead of the deprecated
+entry point `/typo3/index.php`.
+
+Apache configuration before:
+
+..  code-block:: apache
+    :emphasize-lines: 2-4
+
+    RewriteCond %{REQUEST_FILENAME} !-f
+    RewriteCond %{REQUEST_FILENAME} !-d
+    RewriteCond %{REQUEST_FILENAME} !-l
+    RewriteRule ^typo3/(.*)$ %{ENV:CWD}typo3/index.php [QSA,L]
+
+    RewriteCond %{REQUEST_FILENAME} !-f
+    RewriteCond %{REQUEST_FILENAME} !-d
+    RewriteCond %{REQUEST_FILENAME} !-l
+    RewriteRule ^.*$ %{ENV:CWD}index.php [QSA,L]
+
+Apache configuration after:
```

Comment:
How about utilizing that nifty "diff" format you showed me a few weeks back?

Reply:
I think, currently this way it is better readable, we also have code highlighting for apache and nginx here.

Reply:
Ack